### DISCUSSION TITLE
Changing preset colors with IKEA Styrbar

### DISCUSSION THREAD

**Discussion by malaakso** [Koenkk/zigbee2mqtt#9620] [Devices]
I have an IKEA	Trådfri LED1924G9 color bulb and Styrbar (E2001/E2002) remote. Based on my reading I added the bulb to a magic group 65289 in addition to a normal group and bound the remote's OnOff and LevelCtrl clusters to the normal group. Problem is that pressing the arrow buttons seem to cause the bulb to jump over one color (and sometimes not changing color at all).

Styrbar seems to allow binding to the bulb directly as well, in this case the arrow buttons won't do anything. Finally, I bound the remote to both group and bulb, in this case arrow buttons jump over *two* colors.

Anyone has similar experiences? I verified that the remote is good by pairing it directly with the bulb. In this case colors are called sequentially as they should.

I also tried a spare E1524, but the latest FW doesn't bind to group at all, so I couldn't get that to change colors. It also advertises the scenes cluster, but binding or not binding that doesn't seem to make any difference.

**Comment by bjkluft** [comment]
I know this is an old topic but it's the only one describing a more or less issue as I have with mine.
My setup is exactly the same: I have a LED1924G9 bulb and the Styrbar (E2002) remote (which is sold as a set). I've updated the remote to firmware 2.4.5 using this [link](https://github.com/Koenkk/zigbee2mqtt/issues/18515#issuecomment-1742177146)
The remote is bound to the bulb, lik this:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/12735834/b31680bb-b46c-4311-8aaa-2a3bbd5b9a2c)
The light turns on/off and I can lower/raise the brightness with the top and bottom buttons. However the left and right buttons do nothing at all. Within Home Assistant I only see the update status:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/12735834/c3387f85-a664-43bc-91d9-03bca355d235)
I've removed and added the remote from the Zigbee2MQTT but that didn't solve anything.
Something is clearly not correct but I don't seem to be able to figure this out.
All help is appreciated.

Kind regards,
Jeroen

**Reply by malaakso** [reply]
I've concluded that the combination of Styrbar 2.4.5 and LED1924G9 just doesn't work. Styrbar with older FW has the behavior I've described above. And Styrbar 2.4.5 controls colors of LED1624G9 FW 2.3.093 just perfectly. I assume that LED1924G9 with its relatively old firmware still expects binding via group which is not possible with 2.4.5.
